Xabe.FFmpeg/Subtitle.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Threading.Tasks;
+using Xabe.FFmpeg.Enums;
+
+namespace Xabe.FFmpeg
+{
+    /// <summary>
+    ///     Reference to subtitle file
+    /// </summary>
+    public class Subtitle
+    {
+        private readonly string _path;
+
+        /// <inheritdoc />
+        public Subtitle(string path)
+        {
+            _path = path;
+        }
+
+        /// <summary>
+        ///     Convert subtitle to specific format
+        /// </summary>
+        /// <param name="outputPath"></param>
+        /// <param name="subtitleFormat"></param>
+        /// <returns></returns>
+        public async Task<bool> Convert(string outputPath, SubtitleFormat subtitleFormat)
+        {
+            var codec = "";
+            switch(subtitleFormat)
+            {
+                case SubtitleFormat.SRT:
```

Comment:
why not 'format extension' attribute on enum? this switch will be reduced

Reply:
Changed it to description attribute